welcome/welcome.py

```diff
@@ -863,3 +868,12 @@ def __today() -> int:
         """Gets today's date in ordinal form."""
 
         return datetime.date.today().toordinal()
+
+    @staticmethod
+    def format_list(*items, join="and", delim=", "):
```

Comment:
Three things:
1. The method name should start with `__`.
2. I'm not really sure why the first parameter is declared the way it is, especially given that the one place where this is used the argument is originally a list and needs to be unpacked to be passed here, only to be treated as a list again. This can probably just be declared as a plain old list (and the place where it's called doesn't need to be unpacked anymore).
3. All these parameters, and the return, should have type annotations.

Reply:
turns out redbot has a humanize_list function which does the same exact thing as what this does, so I will use that instead!

Reply:
Oh, even better! I didn't know that existed, lol.